src/controllers/AuthController.js

```diff
@@ -0,0 +1,42 @@
+import jwt from 'jsonwebtoken';
+import helpers from '../helpers';
+import services from '../services';
+import findUser from '../services/findUser';
+
+const { forgotPasswordMessage } = helpers;
+const { sendMail } = services;
+
+/**
+ * Auth controller class
+ * @class
+ */
+export default class AuthController {
+  /**
+ *
+ *
+ * @static
+ * @param {object} request
+ * @param {object} response
+ * @returns {json} - json
+ * @memberof AuthController
+ */
+  static async forgotPassword(request, response) {
+    const { email } = request.body;
+    try {
+      const user = await findUser(email, response);
+      const { id, firstName } = user;
+      const token = jwt.sign({ id }, process.env.JWT_KEY, { expiresIn: '2h' });
```

Comment:
I think it will better if you used the callback version of jwt.sign  and send the email in it